filters/awssigner/internal/utils.go

```diff
@@ -0,0 +1,42 @@
+package awssigner
+
+import (
+	"strings"
+)
+
+func StripExcessSpaces(str string) string {
```

Comment:
Please check if this could be replaces by standard functions like https://pkg.go.dev/strings#TrimSpace

Reply:
Took a look, `StripExcessSpaces` also replaces more than one consecutive spaces by just one space without regex. TrimSpace directly may not work.